### PR TITLE
fix(building-rollup): disable http server caching

### PR DESCRIPTION
**Pull request by thepassle**

### CODE REVIEW

File: packages/building-rollup/README.md [hunk 1]
```diff
@@ -68,15 +68,17 @@ To illustrate this, here is an example of scripts which our rollup config will *
...
+    "start:build": "http-server dist -o -c-1",
+    "watch:build": "rimraf dist && rollup --watch -c rollup.config.js & http-server dist -o -c-1",
   }
 }
 ```
 - `build` builds your app and outputs it in your `dist` directory
 - `start:build` runs your built app from `dist` directory
 - `watch:build` builds and runs your app, rebuilding when input files change
 
+Note that caching is disabled by default, if you want to enable caching, remove `-c-1` from the `start:build` and `watch:build` scripts.
```
Comment (LarsDenBakker) [inline packages/building-rollup/README.md]: No need for this note I think. Starts to become too much information. People wont use this in production

Comment (daKmoR) [reply packages/building-rollup/README.md]: true - my bad I suggested it....

Comment (thepassle) [reply packages/building-rollup/README.md]: Removed 👍

**Comment by daKmoR** [comment]
oh we probably should also update the generator for it?

wanna do here or in a separate PR?

**Comment by thepassle** [comment]
@dakmor Good one, fixed it in this PR

**Pull request merged**